build.rs

```diff
@@ -49,3 +50,15 @@ fn compile_sass() {
     let mut file = File::create(&dest_path).unwrap();
     file.write_all(css.as_bytes()).unwrap();
 }
+
+fn copy_js() {
+    use std::io::Read;
+    let mut javascript = String::new();
+    let source_path = Path::new(concat!(env!("CARGO_MANIFEST_DIR"), "/templates/menu.js"));
+    let dest_path = Path::new(&env::var("OUT_DIR").unwrap()).join("menu.js");
+    File::open(&source_path)
+        .expect("open templates/menu.js")
+        .read_to_string(&mut javascript)
+        .expect("read templates/menu.js");
```

Comment:
You can now remove this call to `File::open` and the `javascript` variable.

(sorry I'm being so picky, the PR is great overall)